src/App.js

```diff
@@ -1,13 +1,81 @@
-import React from 'react';
+import React from "react";
+import "./styles.css";
+import TodoForm from "./components/TodoForm";
+import Todo from "./components/Todo";
+// import TodoList from "./components/TodoList";
+let idCounter = () => {
+  let id = 4;
+  return function countUp() {
+    return id++;
+  };
+};
+
+const idUp = idCounter();
+
+const todos = [
+  { task: "Organize Garage", id: 1, completed: false },
+  { task: "Bake Cookies", id: 2, completed: false },
+  { task: "Master React", id: 3, completed: false }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos
+    };
+  }
+
+  removeTodo = id => {
+    this.setState({
+      todos: this.state.todos.filter((todo, index) => todo.id !== id)
+    });
+  };
+
+  handleChange = e => {
+    this.setState({ value: e.target.value });
```

Comment:
4. Right now, `handleChange` doesn't do anything bec you don't have a state that's called `value`.
What you're aiming to update here is `newTask`.
```suggestion
    this.setState({ [e.target.name]: e.target.value });
```